examples/platform/linux/CommissioneeShellCommands.cpp

```diff
@@ -61,6 +62,9 @@ static CHIP_ERROR PrintAllCommands()
     streamer_printf(sout,
                     "  sendudc <address> <port>   Send UDC message to address. Usage: commissionee sendudc 127.0.0.1 5543\r\n");
 #endif // CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONER_DISCOVERY_CLIENT
+    streamer_printf(sout,
+                    "  restartmdns <commissioningMode> <additionalPairing>   Start Mdns with given settings. Usage: commissionee "
+                    "startmdns true true\r\n");
```

Comment:
nit: example seems to be missing a "re".

Reply:
fixed